students/8/main.js

```diff
@@ -39,56 +39,77 @@ const ex6 = () => {
 }
 
 const ex7 = () => {
-    const peopleArray =[];
-    console.table(makepeople(peopleArray));
+    const peopleArray = [];
+    console.log(makepeople(peopleArray));
+}
+
+const ex8 = () => {
+    let personArray = [];
+    console.log(countOver20(personArray))
 }
 
 //
 // Your functions here...
 //
 
+
+//Ex 8
+
+const createPerson = (name, age) => {
+    return {
+        fullName: name,
+        age: age,
+        // ...other properties
+    };
+}
+
+const countOver20 = (personArray) => {
+    let p1 = createPerson("Alice", 11)
+    let p2 = createPerson("Bob", 22)
+    let p3 = createPerson("Charlie", 33)
+
+    personArray.push(p1)
+    personArray.push(p2)
+    personArray.push(p3)
+    //  return personArray;
```

Comment:
Can we delete this code (`//  return personArray;`)?

Reply:
done